templates/generation/release_please_config_generation.sh

```diff
@@ -0,0 +1,171 @@
+#!/bin/bash
+
+set -e
+
+echo "{" > .release-please-manifest.json
+
+echo ""\""google-cloud-gapic-bom"\"": "\""0.0.0"\""," >> .release-please-manifest.json
+
+GENERATION_DIR=$(dirname -- "$0");
+
+num_modules="$(wc -l < ../../repos.txt)"
```

Comment:
Don't rely on repos.txt

Reply:
What should I rely on, then? As a source of truth to calculate the total number of modules in the config files?

Reply:
The number of the modules.

Reply:
Done.